util/FINETUNE_LM_PARAMETERS.md

```diff
@@ -0,0 +1,43 @@
+# Finetune Language Model Parameters: Alpha, Beta
+This topic is to explain how did `finetune_lm_params.py` fit `alpha` and `beta`
+
+## Inspiration
+According to mesh plot result of [Paddle/DeepSpeech-Grid-Search](https://github.com/PaddlePaddle/DeepSpeech/raw/develop/docs/images/tuning_error_surface.png), I assume `BatchWER v.s. Alpha` is `ideal valley parabolic curve`, i.e. which fits this formula below:
+
+Idea Valley Parabola Formula
+===
+- Assume there exists a `lowest WER` satisfies the formula: ![A.1](https://latex.codecogs.com/gif.latex?WER_{batch}=A_0(\\alpha-\\alpha_{best})^2+A_1 "A.1"),  `(A.1)`
+- Where ![](https://latex.codecogs.com/gif.latex?A_0>0) and ![](https://latex.codecogs.com/gif.latex?A_1>0)
+- If we set ![](https://latex.codecogs.com/gif.latex?\\alpha=\\alpha_{best}), the `WER` should be lowest: ![](https://latex.codecogs.com/gif.latex?WER_{lowest}=A_1), so the target is to find out the constant ![](https://latex.codecogs.com/gif.latex?\\alpha_{best})
+- To solve the `best alpha`, I decompose the formula, and change the coefficient as below:
+- ![A.2](https://latex.codecogs.com/gif.latex?WER_{batch}=A_0\\alpha^2-2A_0\\alpha_{best}\\alpha+A_1\\alpha_{best}^2+A_1 "A.2"),  `(A.2)`
+
+Polynomial Idea Valley Parabola Formula
```

Comment:
I don't understand the need for this section. All the results here can be derived directly and more simply from the section above without introducing the B's at all.